sat-plugin/src/main/java/org/openhab/tools/analysis/tools/PmdChecker.java

```diff
@@ -36,13 +36,22 @@
 
     private static final String DEFAULT_RULESET_XML = "rulesets/pmd/rules.xml";
 
+    private static final String DEFAULT_FILTER_XML = "rulesets/pmd/suppressions.properties";
```

Comment:
@lpapazow, I saw the default file that you have added is empty.

I see it is a properties file, but I couldn't find much info how to use it.
Could you provide maybe a link to the official documentation ?